flink/src/main/resources/interpreter-setting.json

```diff
@@ -4,17 +4,17 @@
     "name": "flink",
     "className": "org.apache.zeppelin.flink.FlinkInterpreter",
     "properties": {
-      "host": {
-        "envName": "host",
+      "jobmanager.rpc.address": {
+        "envName": "jobmanager.rpc.address",
         "propertyName": null,
-        "defaultValue": "local",
-        "description": "host name of running JobManager. 'local' runs flink in local mode."
```

Comment:
the flink 1.4.0 use the config 'jobmanager.rpc.address' and 'jobmanager.rpc.port' instead off 'host'  and 'port' to start JobManager; I think the configure is used for flink, so it will be more understandable to use the 'jobmanager.rpc.address' and 'jobmanager.rpc.port'

Reply:
btw, don't you have to change [this line](https://github.com/jianran/zeppelin/blob/e57a2314624b5de34a1af69753ae9e0d6df63378/flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java#L155) and [this line](https://github.com/jianran/zeppelin/blob/e57a2314624b5de34a1af69753ae9e0d6df63378/flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java#L147) and [this line](https://github.com/jianran/zeppelin/blob/e57a2314624b5de34a1af69753ae9e0d6df63378/flink/src/main/java/org/apache/zeppelin/flink/FlinkInterpreter.java#L139) for the new names to work?

actually what I was saying was we should keep both old and new names, just have the new names working in the same lines where the old names are used - this way people with existing config will continue to work. And we can remove the old config names in the next release.

Reply:
ok, I commited the follow code to keep both old and new names